docs/getting_started/install_using_ddev.md

```diff
@@ -0,0 +1,397 @@
+
+By default, DDEV uses ports 80 and 443.
+You can [set different ports](https://ddev.readthedocs.io/en/latest/users/usage/troubleshooting/#method-2-fix-port-conflicts-by-configuring-your-project-to-use-different-ports) with a command like the following:
+
+```bash
+ddev config --http-port=8080 --https-port=8443
+```
+
+### 3. Start DDEV
+
+Proceed by starting your DDEV environment with the following command:
+
+```bash
+ddev start
+```
+
+!!! tip
+
+    If you forgot some part of configuration, you can set it by using `ddev config` later, but afterwards you have to restart DDEV using `ddev restart`.
+
+### 4. Composer authentication
+
+Next, you need to [set up authentication tokens](install_ibexa_dxp.md#set-up-authentication-tokens) by modifying the Composer configuration.
+You must run the following command **after** executing  `ddev start`, because the command will run inside the container.
+
+```bash
+ddev composer config --global http-basic.updates.ibexa.co <installation-key> <token-password>
+```
+
+Replace `<installation-key>` and `<token-password>` with your actual installation key and token password, respectively.
+
+If you want to reuse an existing `auth.json`file, see [Using an auth.json](#using-an-authjson).
+
+### 5. Create Ibexa DXP project
+
+Once DDEV is running, use Composer to create a new Ibexa DXP project. Remember to replace `<edition>` and `<version>` with your desired edition and versions respectively.
+
+```bash
+ddev composer create ibexa/<edition>-skeleton:<version>
+```
+
+### 6. Install the DXP and its database
+
+Once you've made this change, you can proceed to install Ibexa DXP.
+
+```bash
+ddev php bin/console ibexa:install ibexa-<edition>
+ddev php bin/console ibexa:graphql:generate-schema
+```
+
+### 7. Open browser
+
+Once the above steps are completed, open your Ibexa DXP webpage by running the `ddev launch` command.
+
+!!! tip
+
+    You can also see the project URL in the `ddev start` output.
+
+### 8. Start using Ibexa DXP
+
+You can now start using Ibexa DXP and implement your own website on the platform.
+
+You can edit the configuration and code in the DDEV project directory.
+You can use commands listed in the documentation by prefixing them with `ddev exec` or by opening a terminal inside the container using `ddev ssh`.
+For example, if a guideline invites you to run `php bin/console cache:clear`, you can do it in the DDEV container in one of the following ways:
+
+- run `ddev php bin/console cache:clear`
+- enter `ddev ssh` and run `php bin/console cache:clear` after the new prompt
+
+## Other options for configuration
+
+DDEV offers several ways to achieve a same thing, offering different levels of flexibility or adaptability to your development environment.
+
+!!! tip
+
+    Learn more about the [DDEV commands](https://ddev.readthedocs.io/en/latest/users/usage/commands/):
+    
+    - by running [`ddev list`](https://ddev.readthedocs.io/en/latest/users/usage/commands/#list) to list them all,
```

Comment:
@adriendupuis It lists all environments, so maybe it's good to precise in the description that it refers to environments not to listing all commands.

Reply:
@juskora I really meant to list the commands but got confused.

Can you check the following fix proposal, please?

```suggestion
    - by running [`ddev --help`](https://ddev.readthedocs.io/en/latest/users/usage/cli/#using-the-ddev-command) to list them all,
```

Reply:
Yes, it's okay.